src/Features/CSharp/Portable/Completion/KeywordRecommenders/StackAllocKeywordRecommender.cs

```diff
@@ -16,20 +16,41 @@ public StackAllocKeywordRecommender()
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
         {
-            // type t = |
-            var token = context.TargetToken;
-            if (token.IsUnsafeContext())
+            var node = context.TargetToken.Parent;
+
+            if (node == null)
```

Comment:
can't happen.

Reply:
It can. Check `TestNotAtRoot_Interactive`. There won't be anything parsed in an empty string to have a token or a node.

---
In reply to: [155677699](https://github.com/dotnet/roslyn/pull/23655#discussion_r155677699) [](ancestors = 155677699)